event_model/__init__.py

```diff
@@ -119,6 +119,74 @@ def bulk_datum(self, doc):
         self.datum_page(bulk_datum_to_datum_page(doc))
 
 
+
+class Filler:
+    """Pass documents through, loading any externally-referenced data."""
+    ATTEMPTS = 10
+
+    def __init__(self, handler_registry, handler_cache=None, datum_cache=None):
```

Comment:
:+1: for passing the caches in.

Would advocate for kwarg-only, but I think we still have to support py2 on this repo?

Reply:
Actually we do not! Some time over the summer @stuartcampbell gave his blessing for py2-only databroker. And this would not preclude writing a py2-compat data access client anyway. Keyword-only it is....